src/remoteRequest.js

```diff
@@ -0,0 +1,127 @@
+/**
+ * Remote Request question type plugin
+ */
+define([
+    'jquery',
+    'underscore',
+    'vellum/mugs',
+    'vellum/tree',
+    'vellum/util',
+    'vellum/widgets',
+    'vellum/core'
+], function (
+    $,
+    _,
+    mugs,
+    Tree,
+    util,
+    widgets
+) {
+    var REMOTE_REQUEST_TYPE = "RemoteRequest",
+        REMOTE_REQUEST_SECTIONS = [
+            {
+                slug: "main",
+                displayName: gettext("Basic"),
+                properties: [
+                    "nodeID",
+                    "url",
+                    "parameters",
+                    "comment",
+                ],
+            },
+        ],
+        RemoteRequest = util.extend(mugs.defaultOptions, {
+            typeName: gettext('Remote Request'),
+            dataType: REMOTE_REQUEST_TYPE,
+            icon: 'fa fa-globe',
+            isDataOnly: true,
+            isTypeChangeable: false,
+            supportsDataNodeRole: true,
+            writeDataNodeXML: function (xmlWriter, mug) {
+                mug.form.vellum.data.remoteRequest.requestMugs.push(mug);
+            },
+            spec: {
```

Comment:
Can/should there be validation for this mug? Like requiring a URL to be provided?

Reply:
Good idea.